fe-library-app/src/components/AppRouter/pages/LoginPage.tsx

```diff
@@ -48,6 +48,7 @@ const LoginPage = () => {
       .then((response) => {
         sessionStorage.setItem('token', response.data.AccessToken)
         navigation('/home')
+        window.location.reload()
```

Comment:
sto moramo ovo?

Reply:
verovatno sam grekom to postavio sad cu obrisati